src/js/communities/communities.js

```diff
@@ -0,0 +1,38 @@
+import {Communities} from "./communities-data.js";
+const renderArea = document.getElementById("groups");
+Communities.map((val)=>{
+  const {name, location, period, links, demographic, format} = val;
+  const {twitter, facebook, website, meetUp, linkedIn} = links;
+  const {interval, day, weekDay} = period;
+
+  const newHTML = `<h4>
+    <a href=${
+      //uses the group's Meetup link if they don't have a website
```

Comment:
Again SUPER minor but could we polish this up with a space and capital letter. eg.
// Uses the group's Meetup link if they don't have a website

Reply:
Thank you for your feedback. I'll be sure to revise my changes accordingly.